riscv/tests/riscv_data/std_hello_world/README.md

```diff
@@ -0,0 +1,35 @@
+My best shot at compiling this so far is:
```

Comment:
Do we still need this file?

Reply:
No, but it would make an interesting comment in the function that generates the compilation command.